### PR TITLE
Add readme entry about npm usage

### PR DESCRIPTION
**Pull request by stonko1994**
## Description
Since version 3.0.0 we upload the integration to NPM but we don't mention that in the readme

## Solution
This PR updates the readme regarding NPM usage.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Chromecast support 
 - Update content metadata attributes on the receiver directly via the sender instance of the integration
+- Changelog entry about NPM usage
```
Comment (jarhoax) [inline CHANGELOG.md:12]: Are you normally writing `CHANGELOG` or `Changelog`?

And is this best located in the `Added` section? For me that's more an internal thing.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
 
 ## Getting Started
 
+### Installation
+
+#### Using NPM
+Install the npm package:
```
Comment (jarhoax) [inline README.md:9]: Add empty line (or remove it in the other sections)

File: README.md [hunk 3]
```diff
@@ -16,38 +30,71 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
...
     
-    player.load(sourceConfig).then(function() {
-      console.log('player loaded');
-    }, function(reason) {
-      console.error('player setup failed', reason);
-    });
-    ```
-
-4. Release the instance by calling `conviva.release()` before destroying the player by calling `player.destroy()`
+    1. Using custom Build:
```
Comment (jarhoax) [inline README.md:61]:
```suggestion
    2. Using custom Build:
```

File: README.md [hunk 4]
```diff
@@ -16,38 +30,71 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
...
+            };
+            
+            player.load(sourceConfig).then(function() {
+              console.log('player loaded');
+            }, function(reason) {
+              console.error('player setup failed', reason);
+            });
+            ```
+
+1. Release the instance by calling `conviva.release()` before destroying the player by calling `player.destroy()`
```
Comment (jarhoax) [inline README.md:91]:
```suggestion
3. Release the instance by calling `conviva.release()` before destroying the player by calling `player.destroy()`
```

File: README.md [hunk 5]
```diff
@@ -16,38 +30,71 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
...
-      debugLoggingEnabled: true, // optional
-      gatewayUrl: 'https://youraccount-test.testonly.conviva.com', // optional, TOUCHSTONE_SERVICE_URL for testing
-    });
-    
-    var sourceConfig = {
-      // ...
-    };
+1. Include `conviva-core-sdk.min.js` **first of all** in your HTML document
+
+1. Create an instance of `ConvivaAnalytics` **before** calling `player.load(...)` and pass in your Conviva `CUSTOMER_KEY` and optional configuration properties:
```
Comment (jarhoax) [inline README.md:29]:
```suggestion
2. Create an instance of `ConvivaAnalytics` **before** calling `player.load(...)` and pass in your Conviva `CUSTOMER_KEY` and optional configuration properties:
```

**Review by jarhoax — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (stonko1994) [reply CHANGELOG.md:12]: -> `changelog`

We don't have an internal section as this is open source.

File: README.md (hunk 3 above)
Comment (stonko1994) [reply README.md:61]: That's markdown. You don't need to increase the number here. (It should be `1.` everywhere).

That enables that you don't need to change all numbers if you add a new step somewhere in between.

Comment (jarhoax) [reply README.md:61]: Thanks for clarifying this. Didn't know that 😃

**Review by jarhoax — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,11 +1,20 @@
...
 ## Getting Started
+### Installation
+#### Using NPM
+Install the npm package:
+```
+npm i @bitmovin/player-integration-conviva --save-dev
+```
+
+#### Use custom build
+Build the file script by running `npm run build`
```
Comment (jmsn) [inline README.md]: What is meant by `file script`?

File: README.md [hunk 7]
```diff
@@ -15,44 +24,74 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
...
-3. Create an instance of `ConvivaAnalytics` **before** calling `player.load(...)` and pass in your Conviva `CUSTOMER_KEY` and optional configuration properties:
-    ```js
-    var playerConfig = {
-      key: 'YOUR-PLAYER-KEY',
-      // ...
-    };
-
-    var container = document.getElementById('player');
-    var player = new bitmovin.player.Player(container, playerConfig);
+1. Include `conviva-core-sdk.min.js` **first of all** in your HTML document
```
Comment (jmsn) [inline README.md]: Would rephrase to `as first of all scripts` or something

**Review by jmsn — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (stonko1994) [reply README.md]: The JavaSCRIPT file 😬.

What about:
```suggestion
Build the JS file by running `npm run build`
```

Comment (jmsn) [reply README.md]: I'm fine with that

File: README.md [hunk 8]
```diff
@@ -1,11 +1,20 @@
...
 
 ## Getting Started
+### Installation
+#### Using NPM
+Install the npm package:
+```
+npm i @bitmovin/player-integration-conviva --save-dev
+```
+
+#### Use custom build
```
Comment (jmsn) [inline README.md]: To be consistent with line 8

```suggestion
#### Using custom build
```

**Review by jmsn — APPROVED** [review]
